### PR TITLE
Convert SquooshRender.kt

### PR DESCRIPTION
**Pull request by yiqunw700**
Fixes: #1908

### CODE REVIEW

File: designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt [hunk 1]
```diff
@@ -333,12 +332,13 @@ private fun squooshTextRender(
     drawContext.canvas.save()
     drawContext.canvas.translate(0.0f, verticalCenterOffset)
 
-    val strokeWidth = style.nodeStyle.stroke.get().stroke_weight.toUniform() * density.density
+    val strokeWidth =
+        (style.nodeStyle.strokeOrNull?.strokeWeightOrNull?.toUniform() ?: 0f) * density.density
```
Comment (timothyfroehlich) [inline designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt:336]: I think in this case it's better to use the normal proto behavior to return a default values (which is 0) if it wasn't present, like this:
```suggestion
        (style.nodeStyle.stroke.strokeWeight.toUniform()) * density.density
```

What do you think, @rylin?

File: designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt [hunk 2]
```diff
@@ -333,12 +332,13 @@ private fun squooshTextRender(
...
     // Only drop shadow is supported now.
-    val shadow = style.nodeStyle.text_shadow.getOrNull()
-    val blurRadius = (shadow?.blur_radius ?: 0f) * density.density
-    val xOffset = (shadow?.offset_x ?: 0f) * density.density
-    val yOffset = (shadow?.offset_y ?: 0f) * density.density
+    val shadow = style.nodeStyle.textShadowOrNull
+    val blurRadius = (shadow?.blurRadius ?: 0f) * density.density
+    val xOffset = (shadow?.offsetX ?: 0f) * density.density
+    val yOffset = (shadow?.offsetY ?: 0f) * density.density
```
Comment (timothyfroehlich) [inline designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt:341]: Should be able to handle this with proto returning defaults:

```suggestion
    val shadow = style.nodeStyle.textShadow
    val blurRadius = shadow.blurRadius * density.density
    val xOffset = shadow.offsetX * density.density
    val yOffset = shadow.offsetY * density.density
```

**Review by timothyfroehlich — APPROVED** [review]

**Review by timothyfroehlich — APPROVED** [review]

File: designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt (hunk 1 above)
Comment (rylin8) [reply designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt:336]: It's a bit cleaner to use the default values, but using the `orNull` fields make it explicitly clear that those values can be null...

Comment (rylin8) [reply designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt]: Done

File: designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt (hunk 2 above)
Comment (rylin8) [reply designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt]: Done

**Pull request merged**